tools/changelogger/src/render.rs

```diff
@@ -181,8 +188,8 @@ fn render_entry(entry: &HandAuthoredEntry, mut out: &mut String) {
         .map(|t| t.to_string())
         .chain(entry.references.iter().map(to_md_link))
         .collect::<Vec<_>>();
-    if !maintainers().contains(&entry.author.to_ascii_lowercase().as_str()) {
-        references.push(format!("@{}", entry.author.to_ascii_lowercase()));
+    if !is_maintainer(&entry.author) {
+        references.push(format!("@{}", entry.author));
     };
```

Comment:
Why were we converting author names to lowercase here? 
Wouldn't it make the code simpler (and maybe even "more correct") if we make this comparison case-sensitive?

Reply:
I think the casing doesn't matter because my GitHub name is capitalized but it looks like these are all equivalent:
- @veLFi
- @Velfi
- @velfi

raw markdown:
```
- @veLFi
- @Velfi
- @velfi
```